src/etc/inc/certs.inc

```diff
@@ -2321,7 +2321,7 @@ function cert_notify_expiring() {
 		/* Fetch and analyze expiration */
 		$expiredays = cert_get_lifetime($cert, true);
 		/* If the result is null, then the lifetime data is missing, so skip the invalid entry. */
-		if ($expiredays == null) {
```

Comment:
Loose comparison on the last day before certificate expiration does not generate an alert.